frontend/src/pages/Trips.jsx

```diff
@@ -47,7 +48,7 @@ export default function Trips() {
   // DELETE
   const handleDelete = async (id) => {
     try {
-      const response = await fetch(`https://0nkryc0lmb.execute-api.us-east-1.amazonaws.com/deleteTrip?tripId=${id}`, {
+      const response = await fetch(API_Endpoint + 'deleteTrip?tripId=${id}', {
```

Comment:
I think it would be neater and work better to use backticks for these API calls instead of concatenating with the `+`. 
I don't think that the ${} syntax works when a string is enclosed in regular quotes.
So this should be
```suggestion
      const response = await fetch(`${API_Endpoint}deleteTrip?tripId=${id}`, {
```

Reply:
Thanks! I updated accordingly